File: androidApp/src/main/kotlin/com/malaria/android/ui/screens/ProvisioningIncompleteView.kt
package com.malaria.android.ui.screens

import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.foundation.layout.padding
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.unit.dp

/**
 * Shown when the device has been admin-provisioned but no microscopist has
 * claimed it yet (spec §10, Phase 2 of onboarding). Phase 7 implements the
 * claim flow.
 */
@Composable
fun ProvisioningIncompleteView() {
    Box(
        modifier = Modifier
            .fillMaxSize()
            .padding(start = 16.dp, end = 16.dp),
        contentAlignment = Alignment.Center,
    ) {
        Text("Provisioning incomplete — Phase 8")
    }
}

File: iosApp/Views/ProvisioningIncompleteView.swift
import SwiftUI

/// Shown when the device has been provisioned by an admin but the
/// clinician profile has not yet been claimed. Phase 6 routes this
/// to the microscopist-onboarding flow. Per spec §10.
struct ProvisioningIncompleteView: View {
    var body: some View {
        VStack(alignment: .leading, spacing: 16) {
            Text("Provisioning incomplete")
                .font(.title)
            Text("This device is awaiting clinician onboarding. Phase 6 wires the claim flow.")
                .foregroundStyle(.secondary)
            Spacer()
        }
        .padding(.leading, 24)
        .padding(.trailing, 24)
        .padding(.top, 24)
        .frame(maxWidth: .infinity, alignment: .leading)
    }
}
